code/Test_definitions/Traffic_Influence.yaml

```diff
@@ -0,0 +1,1030 @@
+    Demand). Providing the optimal routing is indeed an important step to get
+    the optimal latency.\
+    If the TI API is used to get the best routing at the Edge for a Device in a
+    geographical location and the Device moves to another geographical location,
+    the TI API can be invoked to get the optimal routing in the new geographical
+    location for that Device.
+    ## Introduction
+    The TI API provides the capability to establish the best routing, in terms
+    of latency, in a specific geographical area, between the user Device, e.g.
+    the user’s smartphone, and the optimal EAS instance nearby. If the Device
+    moves in a different geographical location where a more suitable EAS
+    instance is available, the TI API can be invoked to influence the Device
+    connectivity to get the optimal routing to the that local instance. It is i
+    mportant to notice that it is a task of the Application invoking the TI API
+    to detect the changes in the Device location.\
+    The generic architecture for the Service can foresee some Cloud instances of
+    the Application, one or more Edge Instances of the Application. A component
+    of the Service is the TI API Consumer. This logical component can be
+    integrated in other components of the Service to invoke the TI API, creating
+    a "TrafficInfluence" resource specifying the desired intent.\
+    The TI API Producer implements the intent specified in the
+    "TrafficInfluence" resource.\
+    While the TI API can be invoked to activate the fastest routing for any
+    user, it can also be used to request the best routing for a specific user.
+    Invoking the TI API for each user, many "TrafficInfluence" resources are
+    created for each user to provide the requested routing for a set of users.\
+    The same approach is used for the geographical locations where the influence
+    of the traffic must be applied. Invoking the TI API without specifying a
+    geographical area activates the optimal routing toward any EAS instance,
+    while invoking the TI API specifying a geographical area activates the
+    optimal routing only toward the EAS instance located closest to that
+    geographical area.\
+    To activate the optimal routing in selected geographical areas, the TI API
+    must be invoked for each geographical area.\
+    The TI API can be used to:
+    - optimise the routing when Devices need to consume the service provided
+        by a local EAS Instances.
+    - affect an already established Device routing when the Device moves
+        among different geographical locations. When the TI API consumer detects
+        a Device has entered a geographical location where an EAS instance is
+        available, it can invoke the TI API to get the optimal routing toward
+        that EAS instance.
+        If the Device moves to another geographical location, served by another
+        EAS instance, the routing might not be optimal anymore for the new EAS
```

Comment:
I think "the routing might not be optimal anymore for the new EAS" in this statement the "new EAS" is incorrect. It could be the existing serving EAS or current EAS which may not be suitable in new location of the user.